nbs/index.ipynb

```diff
@@ -7,7 +7,13 @@
    "outputs": [],
```

Comment:
<ol><li>instead of os.path prefer pathlib Path</li><li>It looks like you didn't follow the installation steps in nbdev. Usually pip install -e '.[dev]' should be enough</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MeherKharbachi/betting_env/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>